dags/operators/email_schema_change_operator.py

```diff
@@ -0,0 +1,69 @@
+import boto3
+import logging
+import re
+from airflow.exceptions import AirflowException
+from airflow.operators.email_operator import EmailOperator
+from airflow.utils.decorators import apply_defaults
+from difflib import unified_diff
+from os import environ
+
+
+class EmailSchemaChangeOperator(EmailOperator):
+    """
+    Execute a Spark job on EMR and send email on schema changes.
+
+    :param latest_schema: The name of the job.
+    :type latest_schema: string
+
+    :param previous_schema: The e-mail address of the user owning the job.
+    :type previous_schema: string
+
+    and all EmailOperator params except html_content
+    """
+    template_fields = EmailOperator.template_fields + ('latest_schema', 'previous_schema')
+    s3_regex = re.compile('([Ss]3://)?(?P<Bucket>[^/]*)/(?P<Key>.+)')
+    latest_schema = ''
+    previous_schema = ''
+
+    # s3 buckets from env
+    spark_bucket = environ['SPARK_BUCKET']
+    airflow_bucket = environ['AIRFLOW_BUCKET']
+    private_output_bucket = environ['PRIVATE_OUTPUT_BUCKET']
+    public_output_bucket = environ['PUBLIC_OUTPUT_BUCKET']
+    artifacts_bucket = environ['ARTIFACTS_BUCKET']
```

Comment:
These are no longer used in the operator, correct?

Reply:
private_output_bucket is used to define the schema location